src/component/CodeCardBox.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {Text, FlatList, Image, View} from 'react-native';
+
+const PCodeCardBox = props => {
+  const {img, codinglang} = props;
+
+  return (
+    <View style={style}>
```

Comment:
add this style to param and change name incomeStyle
then create one style for this component with name : style/flatListStyle.js 
 then import style from './style/flatListStyle.js '

and styling must be like this : 

<View style={[style.text , incomeStyle.style.text]}>